tests/Unit/PackageTest.php

```diff
@@ -60,4 +63,12 @@ public function testPackageHasFormattedTrackingCode()
         $this->assertEquals(str_replace(' ', '', $tracking_code), $package->getTrackingCode());
         $this->assertEquals(Carrier::CODE_USPS, $package->getCarrierCode());
     }
+
+    public function fedexProvider()
+    {
+        return [
+            ['778890188810'],
+            ['999999999999999'],
```

Comment:
Is this a real 15 digit FedEx code? Would be good to use a valid number in the test.